_drafts/oryx/setup.md

```diff
@@ -0,0 +1,44 @@
+# Setup
+
+This guide explains how to set up your environment for Oryx framework development.
+
+## Prerequisites
+
+To use the Oryx framework, you should be familiar with the following:
+
+- Javascript
+- CSS
+- HTML
```

Comment:
This is about installation, and those technologies are not required for someone who wants to install it (e.g. devops guy, business analyst, etc.)

Reply:
So, let's drop js, css, html here. we'll discuss that in parts where we talk about customisations.